libcontainer/container_linux.go

```diff
@@ -189,6 +202,12 @@ func (c *linuxContainer) Start(process *Process) error {
 		}
 		return newSystemError(err)
 	}
+	// generate a timestamp indicating when the container was started
+	c.creationTS = time.Now().UTC()
+
+	// use the current working directory as the bundle for user output
+	// see runc list command for an example that outputs the original Bundle dir
+	c.creationDir, err = os.Getwd()
```

Comment:
For things like docker it will not.  Can we maybe leave this one out and i'll think about how we want to handle this in libcontainer?